### PR TITLE
ISO date format support

### PR DESCRIPTION
**Pull request by RichAyotte**
ISO date format based off the code from https://github.com/mickhansen/graphql-sequelize/issues/166

### CODE REVIEW

File: src/typeMapper.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import {
-   GraphQLInt,
```
Comment (RichAyotte) [inline src/typeMapper.js]: Editor cleanup up the off by one space invalid indentation in the import.

**Comment by mickhansen** [comment]
This will likely have to be considered a breaking change right?
Remove yarn.lock

**Comment by RichAyotte** [comment]
I'd definitely consider this a breaking change. Sorry about the yarn.lock, new PR coming.

File: src/typeMapper.js [hunk 2]
```diff
@@ -68,12 +69,15 @@ export function toGraphQL(sequelizeType, sequelizeTypes) {
   if (sequelizeType instanceof FLOAT ||
       sequelizeType instanceof DOUBLE) return GraphQLFloat;
 
+  if (sequelizeType instanceof DATE ||
+      sequelizeType instanceof DATEONLY) {
```
Comment (mickhansen) [inline src/typeMapper.js]: is DATEONLY parsed as a date objecti n Sequelize? Isn't it just a date string?

**Comment by mickhansen** [comment]
The new DateType should be exported via src/index

File: src/typeMapper.js (hunk 2 above)
Comment (RichAyotte) [reply src/typeMapper.js]: I think it depends on the version and the latest is String. See https://github.com/sequelize/sequelize/issues/4858 I'll switch it back String to be on the safe side.

**Comment by RichAyotte** [comment]
Thanks for the review and quick merge!

**Comment by mickhansen** [comment]
https://github.com/mickhansen/graphql-sequelize/releases/tag/v9.0.0

**Comment by darkrift** [comment]
The import is bad, the file name is "dateType" not DateType. I installed the module today and it failed because of that.

Error: Cannot find module './types/DateType'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/rlavoie/work/kerozen/acry/backend/node_modules/graphql-sequelize/lib/index.js:17:13)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/rlavoie/work/kerozen/acry/backend/schema/types/root_query_type.js:2:20)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/rlavoie/work/kerozen/acry/backend/schema/index.js:2:23)
    at Module._compile (module.js:652:30)

**Comment by RichAyotte** [comment]
PR is in the queue. https://github.com/mickhansen/graphql-sequelize/pull/622

**Comment by darkrift** [comment]
Hopefully @mickhansen will merge it quickly.

**Pull request merged**